docs/source/bond_family_guide/bond_workload_simulator.rst

```diff
@@ -0,0 +1,179 @@
+..
+   INTEL CONFIDENTIAL
+   Copyright(c) 2016 Intel Corporation All Rights Reserved.
```

Comment:
This header is incorrect. The year should be 2017, and this information is being open-sourced, so it is no longer confidential. There are ~50 files in this PR with this header, and a handful with no header. It looks like most of our doc files _don't_ have a header for whatever reason, but I would get confirmation about whether or not they are necessary, and then make sure that every file in this PR has the same thing: either the right header, or no header.